science/netcdf-fortran/Portfile

```diff
@@ -69,6 +70,11 @@ post-patch {
     }
 }
 
+# Fix for 10.7 copied from:
+# https://github.com/william-dawson/NTPoly/issues/192
```

Comment:
That NTPoly issue describes a build problem. I assume it is the same here and that you are fixing this problem shown in the 10.7 buildbot logs:

```
<stdin>:1219:2: error: invalid instruction mnemonic 'cvttss2sil'
        cvttss2sil      %xmm0, %eax
        ^~~~~~~~~~
<stdin>:1329:2: error: invalid instruction mnemonic 'cvttss2sil'
        cvttss2sil      %xmm0, %eax
        ^~~~~~~~~~
<stdin>:49073:2: error: invalid instruction mnemonic 'cvttss2sil'
        cvttss2sil      %xmm0, %eax
        ^~~~~~~~~~
make[1]: *** [netcdf4.lo] Error 1
```

If so, then increasing the port's revision is not appropriate. [The Guide](https://guide.macports.org/#reference.keywords) explains when and when not to increase the revision.

Reply:
Yes, that buildbot error is what I am trying to fix, approximately the same as in the NTPoly issue.  I will now update to cancel that rev bump.